pkg/tempopb/tempo.proto

```diff
@@ -163,6 +163,12 @@ message Trace {
 message PushResponse {
 }
 
+message IngesterPushResponse {
+  repeated uint32 maxLiveErrorTraces = 1;
```

Comment:
we can just add these fields to the existing `PushResponse`. changing the response type will change the method signature which i think will break Tempo on rollout. 

speaking of, we will need to be thoughtful about what a rollout of this change will look like. we will need to nail down some more of the behavior as discussed in the comments first.